crates/hotshot/task-impls/src/network.rs

```diff
@@ -587,6 +588,10 @@ impl<TYPES: NodeType, V: Versions> NetworkMessageTaskState<TYPES, V> {
                             }
                             HotShotEvent::VidShareRecv(sender, convert_proposal(proposal))
                         },
+                        DaConsensusMessage::VidDisperseMsg3(proposal) => {
+                            // TODO(Chengyu): consensus version check
```

Comment:
curious, why do we need to check consensus version here? given that it's already behind the `VidDispersedMsg3` arm?